src/services/llmService.ts

```diff
@@ -37,20 +37,18 @@ export const extractFlashcards = async (
     if (!config.baseUrl) {
       throw new Error('API base URL is not configured. Please check your environment variables.');
     }
-    const isProxyRequired = true;
+    const isProxyRequired = needsCORSproxy(config.baseUrl);
```

Comment:
What caused us to need this? Seems like an interesting detail, maybe the kind of thing you could include in notes on the PR